src/app/msg-sent/msg-sent.component.css

```
@@ -0,0 +1,17 @@
+#graphic {
+    /* Paper plane SVG */
+    width: 25em;
+}
+
+#buttons > button {
+    /* For each button */
+    margin: 0 1em;
+    margin-top: 2em;
+    font-size: 1.1em;
+}
+
+#title-text {
+    /* "Your friend will receive your message soon!" */
+    text-align: center;
+    font-size: 2em;
+}
```

No newline at end of file
```suggestion
}

```